src/styles/components/Header.scss

```diff
@@ -0,0 +1,9 @@
+.header {
+  padding: 6rem 0.5rem 0 10rem;
+
+  &__logo {
+    min-width: 12rem !important;
+    position: relative !important;
+    height: 100% !important;
+  }
+}
```

Comment:
So the reason I have the `!important` flag here is because Next.js wanted me to use their Image component instead of <img> element. However, the Image component throws this error if I don't put a "width" and "height" property directly on the componen
![Screen Shot 2022-10-05 at 6 34 58 PM](https://user-images.githubusercontent.com/43832671/194183956-95421214-69d3-49e2-bf6b-422c71dd822e.png)
t:

Adding those properties broke my styles so I overrode them with `!important`.